locale/en-GB.json

```diff
@@ -2,5 +2,6 @@
   "decimal": ".",
   "thousands": ",",
   "grouping": [3],
-  "currency": ["£", ""]
+  "currency": ["£", ""],
+  "currencyAbbreviations": ["", "K", "M", "B", "T"]
```

Comment:
From [Language Matters: Millions, Billions and Other Large Numbers](https://www.druide.com/en/reports/millions-billions-and-other-large-numbers)

>The most commonly seen short forms for thousand, million, billion and trillion in North America and the United Kingdom, respectively, are outlined in the table below.

[![image](https://user-images.githubusercontent.com/68416/85064238-43826680-b179-11ea-9b3a-fd16f0071670.png)](https://www.druide.com/en/reports/millions-billions-and-other-large-numbers)

>Of the style guides that address spacing in this context, most (AP Stylebook, Chicago Manual of Style, Canadian Style, The Economist Style Guide) say to leave no space (100bn, for example), although it is also common in books and newspaper articles to see the abbreviation preceded by a space.

Might we consider changing `locale/en-GB.json` to match these?

Reply:
changed in #96

Reply:
Thanks for adding these. I removed them from this PR originally (https://github.com/d3/d3-format/pull/81/commits/7ee5a2c9756b5830aeb1666f36a27dc6369747d6) because they are not indicated in the CDLR, but I think they should be!